MdeModulePkg/Core/Pei/PeiMain/PeiMain.c

```diff
@@ -323,6 +323,21 @@ PeiCore (
       //
       OldCoreData->PeiMemoryInstalled = TRUE;
 
+      if (PcdGetBool (PcdMigrateTemporaryRamFirmwareVolumes)) {
+        DEBUG ((DEBUG_VERBOSE, "Early Migration - PPI lists before temporary RAM evacuation:\n"));
+        DumpPpiList (OldCoreData);
+

```

Comment:
Why is this extra debug info required?  How is it related to this feature?

Reply:
We are printing the PPI list before and after migration, the same way as it is done in default migration case. This dump helps to understand if all the PPIs for a given FV has been migrated to permanent memory and is also helpful for debugging.